pom.xml

```diff
@@ -100,6 +101,7 @@
             <classpathDependencyExclude>ch.qos.logback:logback-classic</classpathDependencyExclude>
             <classpathDependencyExclude>org.slf4j:slf4j-reload4j</classpathDependencyExclude>
             <classpathDependencyExclude>ch.qos.reload4j:reload4j</classpathDependencyExclude>
+            <classpathDependencyExclude>xml-apis:xml-apis</classpathDependencyExclude>
```

Comment:
What is the purpose here to exclude xml-apis here?

Reply:
It was a redundant addition for us as we moved to JDK 11. JDK 11 has the xml-apis included in it. The Xerces jar has this as the compile dependency and had an issue for multiple sources. So I excluded this.

Reply:
xml-apis is not in the pom file as a dependency but has an entry in bindist files for mlcp and mapr. I can remove it from the bindist files